lib/Cli.php

```diff
@@ -583,7 +583,7 @@ protected function serializeComponent(Component $vObj) {
 
         };
 
-        $children = $vObj->children();
+        $children = iterator_to_array($vObj->children());
```

Comment:
In my benchmark I mostly tested things that use things like property-access. A lot of statements like this `$vcalendar->VEVENT->DTSTART` use (in this PR) multiple generators.

I was hoping there would be a fair bit of win there. Changing the iterator_to_array and ksort statements only really help with serializing, which is something I haven't really tested for. I suspect though that the win will be minimal too...